### PR TITLE
ST-769 Close case types

### PR DESCRIPTION
**Pull request by TimPickmanHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-769

### Change description ###
Add several different types of closing case

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerChooseOutcome.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+@Slf4j
+public class CaseworkerChooseOutcome implements CCDConfig<CaseData, State, UserRole> {
+    public static final String CASEWORKER_CLOSE_THE_CASE = "caseworker-choose-the-outcome";
+
+    private static final CcdPageConfiguration closeCaseWarning = new CloseCaseWarning();
+    private static final CcdPageConfiguration closeCaseReasonSelect = new CloseCaseReasonSelect();
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerChooseOutcome.java:34]: Remove the whitespace?

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCloseTheCase.java [hunk 2]
```diff
@@ -29,20 +27,20 @@ public class CaseworkerCloseTheCase implements CCDConfig<CaseData, State, UserRo
     public static final String CASEWORKER_CLOSE_THE_CASE = "caseworker-close-the-case";
 
     private static final CcdPageConfiguration closeCaseWarning = new CloseCaseWarning();
-    private static final CcdPageConfiguration CloseCaseReasonSelect = new CloseCaseReasonSelect();
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCloseTheCase.java:31]: Remove the whitespace change?

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCloseTheCase.java [hunk 3]
```diff
@@ -29,20 +27,20 @@ public class CaseworkerCloseTheCase implements CCDConfig<CaseData, State, UserRo
...
 
     @Override
     public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
 
         var pageBuilder = closeCase(configBuilder);
         closeCaseWarning.addTo(pageBuilder);
-        CloseCaseReasonSelect.addTo(pageBuilder);
+
     }
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCloseTheCase.java:38]: As above?

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by muradalamv1 — APPROVED** [review]

**Pull request merged**